ui/package.json

```diff
@@ -88,7 +88,7 @@
     "clean-webpack-plugin": "^3.0.0",
     "cross-env": "^5.2.0",
     "css-loader": "^3.1.0",
-    "cypress": "3.2.0",
+    "cypress": "^3.8.1",
```

Comment:
this part of the diff should go away when https://github.com/influxdata/influxdb/pull/16401 gets merged.